### ISSUE TITLE
Datatable filter accessibility (DAP) violations 

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#6024]
## Environment

> Operating system
Mac OS Catalina 10.15.3 (19D76)
> Browser
Chrome Version 80.0.3987.87

> Automated testing tool and ruleset
DAP -- IBM Accessibility January 2020
> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?
cloud pal- "1.125.0"

> What did you expect to happen?
No dap voilation
> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?
Use grouping roles or elements to identify related form controls

## Steps to reproduce the issue

1. Step one
Open teh following url
2. Step two
Cick on findings type dropdown
3. Step three
Execute dap
4. etc.

## Additional information

- Screenshots or code
- Notes
Reported to cloud pal routed by them to carbon team

**Comment by emyarod** [comment]
closing as a duplicate of #6023
